### ISSUE TITLE
DBeaver: The generated HTML files using Open With -> Web Browser specifies the wrong character set

### ISSUE THREAD

**Issue by pdanie** [dbeaver/dbeaver#5050]
DBeaver - **Version 5.3.3 CE** - **EA 19-Jan-2019 14:00**
DBeaver driver: **MS SQL Server  / Microsoft Driver** _(the new and old Microsoft Driver)_
Operating System: **Windows 7 / Windows 8.1 / Windows 10**
Database Server: **Microsoft SQL Express 2014, 2016, 2017**

When generating a HTML page from selected result grid data, the HTML file specifies the wrong character setting.

**o** Start _DBeaver_ and connect to a _MS SQL SERVER_ database of your choice.

**o** Execute the SQL statements below.
```
CREATE TABLE TestCharSet(Id INTEGER, Message_1 VARCHAR(64), Message_2 NVARCHAR(64));

INSERT INTO TestCharSet
VALUES (1, 'ABC abc ÅÄÖ åäö', 'ABC abc ÅÄÖ åäö'),
       (2, 'ABC abc ÜÉ üé', 'ABC abc ÜÉ üé');

SELECT * FROM TestCharSet;
```
**o** In the result data grid, select the grid rows and columns and right click and choose **Open With** -> **Web browser**.

![2019-01-19 20-01-09](https://user-images.githubusercontent.com/31191374/51431154-3cbe8f80-1c25-11e9-9f9f-b5153bc6d1c8.png)

**o** In the shown web page, the swedish characters if presented wrong.

![2019-01-19 20-04-18](https://user-images.githubusercontent.com/31191374/51431173-89a26600-1c25-11e9-9768-513fdc79b4f2.png)

**o** Save the web file (in the web browser), and edit the HTML file changing the character set to **UTF-8**.

**o** Open the edited HTML file in your web browser.

![2019-01-19 20-06-25](https://user-images.githubusercontent.com/31191374/51431197-d6863c80-1c25-11e9-856a-cb4e47d33df7.png)

**o** Now the swedish characters are displayed correctly.

I have tried it with **MS SQL Server** (new and old driver), **PostgreSQL**, **MariaDB** and **MS Access**. They all fail to show the corect swedish characters.

Currently _DBeaver_ generate this HTML header (specifies **windows-1252** character set):
```
<html>
  <head>
    <meta http-equiv="content-type" content="text/html; charset=windows-1252">
    ....
    </head>
    <body>
        ...
    </body>
</html>  
```

In my case _DBeaver_ ought to generate this HTML header (specifies **utf-8** character set):
```
<html>
  <head>
    <meta http-equiv="Content-Type" content="text/html; charset=utf-8">
    ....
    </head>
    <body>
        ...
    </body>
</html>  
```

**Comment by serge-rider** [comment]
That's quite odd. Both export ot HTML and Open in web browser use the same code.
Could you test this in EA version?

**Comment by pdanie** [comment]
I have tried this in **EA 29-Jan-2019 20:26** and this works **OK**. My guess when you fixed case #5058 it fixed this misbeaviour as well. **Great Work**
